src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.c

```diff
@@ -8,8 +8,14 @@ void EnTuboTrap_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnTuboTrap_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnTuboTrap_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnTuboTrap_Draw(Actor* thisx, GlobalContext* globalCtx);
+void func_80931004(EnTuboTrap *this, GlobalContext *globalCtx);
+void func_80931138(EnTuboTrap* this, GlobalContext* globalCtx);
+void func_809311C4(EnTuboTrap* this, GlobalContext* globalCtx);
+
+s32 D_80931410[] = { // Colliderinit
+    0x0A110900, 0x20010000, 0x00000000, 0xF7CFFFFF, 0x00040000, 0xF7CFFFFF, 0x00000000, 0x01010000, 0x000B001C, 0x00000000, 0x00000000, 
+};
```

Comment:
This should be converted into a struct, specifically a `ColCylinderInit`.
There's a tool coming in #58 that should help, so you might want to wait for that (which btw with change the type name).